### PR TITLE
Add Scout-App.app v.2.12.12

### PR DESCRIPTION
**Pull request by gobinathm**
- Including new cask for Scout-App - The simplest Sass processor

*If there’s a checkbox you can’t complete for any reason, that's okay, just explain in detail why you weren’t able to do so.*

After making all changes to the cask:

- [x] `brew cask audit --download {{cask_file}}` is error-free.
- [x] `brew cask style --fix {{cask_file}}` reports no offenses.
- [x] The commit message includes the cask’s name and version.

Additionally, **if adding a new cask**:

- [x] Named the cask according to the [token reference].
- [x] `brew cask install {{cask_file}}` worked successfully.
- [x] `brew cask uninstall {{cask_file}}` worked successfully.
- [x] Checked there are no [open pull requests] for the same cask.
- [x] Checked the cask was not already refused in [closed issues].
- [x] Checked the cask is submitted to [the correct repo].

[token reference]: https://github.com/caskroom/homebrew-cask/blob/master/doc/cask_language_reference/token_reference.md
[open pull requests]: https://github.com/caskroom/homebrew-cask/pulls
[closed issues]: https://github.com/caskroom/homebrew-cask/issues?q=is%3Aissue+is%3Aclosed
[the correct repo]: https://github.com/caskroom/homebrew-cask/blob/master/doc/development/adding_a_cask.md#finding-a-home-for-your-cask

### CODE REVIEW

File: Casks/scout.rb [hunk 1]
```diff
@@ -0,0 +1,13 @@
+cask 'scout' do
+  version '2.12.12'
+  sha256 '33a59da631d45fe4abc4b2525cf8148c722eb3d87787a94c0eeee7d81be9b4b7'
+
+  # github.com/scout-app/scout-app was verified as official when first introduced to the cask
+  url 'https://github.com/scout-app/scout-app/releases/download/v2.12.12/OSX_Scout-App_2.12.12.zip'
```
Comment (neutric) [inline Casks/scout.rb]: To make future updates easier, don't hardcode version numbers and use interpolation instead: `url 'https://github.com/scout-app/scout-app/releases/download/v#{version}/OSX_Scout-App_#{version}.zip'`.

File: Casks/scout.rb [hunk 2]
```diff
@@ -0,0 +1,13 @@
...
+  appcast 'https://github.com/jnordberg/irccloudapp/releases.atom',
```
Comment (neutric) [inline Casks/scout.rb]: The URL should be `https://github.com/scout-app/scout-app/releases.atom`. Don't forget to update the checkpoint accordingly.

**Review by neutric — CHANGES_REQUESTED** [review]
Thanks for your contribution, @gobinathm!

**Comment by neutric** [comment]
Maintainers: I am unsure whether this cask should be named `scout`or `scout-app`.

**Comment by gobinathm** [comment]
@neutric The cask name was chosen as per token suggested by running `$(brew --repository)/Library/Taps/caskroom/homebrew-cask/developer/bin/generate_cask_token" '/Applications/Scout-App.app/'`

**Comment by miccal** [comment]
The Cask token `scout` is correct, but please make the other changes as suggested by @neutric

**Comment by gobinathm** [comment]
@miccal @neutric  any thoughts on the below error after updating the URL?

```
brew cask audit Casks/scout.rb --download
Error: Cask 'scout' definition is invalid: 'url' stanza failed with: bad URI(is not URI?): https://github.com/scout-app/scout-app/releases/download/v#{version}/OSX_Scout-App_#{version}.zip
```

**Comment by miccal** [comment]
@gobinathm you must surround the `url` with double-quotes, like so:
```
`url "https://github.com/scout-app/scout-app/releases/download/v#{version}/OSX_Scout-App_#{version}.zip"`
```

**Comment by gobinathm** [comment]
@miccal already did that & code updated..

@neutric can you pls check it again.

**Review by miccal — APPROVED** [review]

**Comment by gobinathm** [comment]
@miccal do we still need "awaiting user reply' label ?

**Comment by neutric** [comment]
@gobinathm Sorry for being sloppy regarding the `url` stanza! Glad you got it sorted out.

**Pull request merged**